vue-components/src/components/Lookup.vue

```diff
@@ -75,9 +78,12 @@ export default Vue.extend( {
 			type: String,
 			default: '',
 		},
+		/**
+		 * The selected menu item, can be of type `MenuItem` or `null`.
+		 */
 		value: {
```

Comment:
Why is this a prop? Also, it seems to be unused?

Reply:
Agree, doesn't seem to be used. It is most likely a leftover from using the TextInput component as a boilerplate.

Reply:
"Currently this prop is not used and does nothing." :( Hahah, poor prop. 

Looks good. Will approve.